manifests/o/OpenJS/NodeJS/Nightly/19.0.0/OpenJS.NodeJS.Nightly.installer.yaml

```
@@ -16,12 +16,12 @@ Commands:
 - npm
 Installers:
 - Architecture: x64
-  InstallerUrl: https://nodejs.org/download/nightly/v19.0.0-nightly20220810ec444037cd/node-v19.0.0-nightly20220810ec444037cd-x64.msi
-  InstallerSha256: 4EB12BF423439CD334668914CCB596461E8B89476D193F3D8D9DCC92557F834C
-  ProductCode: '{900EB62B-0052-4ADB-ADC9-7F40E6F0FC17}'
+  InstallerUrl: https://nodejs.org/download/nightly/v19.0.0-nightly20220811c3770536bc/node-v19.0.0-nightly20220811c3770536bc-x64.msi
+  InstallerSha256: 20CA9074F55C7B65CC42A22D3CC1B73C67B05EA19CA2E7BF8743F141F54AAE56
+  ProductCode: '{1BEC3B45-0547-4212-8A44-CB723E8E6C97}'
 - Architecture: x86
-  InstallerUrl: https://nodejs.org/download/nightly/v19.0.0-nightly20220810ec444037cd/node-v19.0.0-nightly20220810ec444037cd-x86.msi
-  InstallerSha256: B3985290572F565FA9F159DE36363FD838B8364D91986F739296A98EF403F548
-  ProductCode: '{497C9CEA-FC3F-46E1-BFAF-63FFEC2FE8B7}'
+  InstallerUrl: https://nodejs.org/download/nightly/v19.0.0-nightly20220811c3770536bc/node-v19.0.0-nightly20220811c3770536bc-x86.msi
+  InstallerSha256: EB5B34C3DED79B777A15B71BFFD5E521FC7925D3097CEAC58F88CF52D23631F4
+  ProductCode: '{AAA21A1B-FA03-4FB1-A41A-8599E9969150}'
```

```suggestion
  ProductCode: '{1BEC3B45-0547-4212-8A44-CB723E8E6C97}'
```
I get a different product code when I test?